packages/server/modules/core/services/users.js

```diff
@@ -51,16 +41,6 @@ module.exports = {
     return role
   },
 
-  /**
-   * @deprecated {Use changePassword()}
```

Comment:
So turns out the update & validate TS fns in `management.ts` are unused. So what I did was I converted those to fit the behavior & API of these existing ones, and replaced all usages of these old JS ones with the new ones.

While that's a bigger refactor than we usually attempt in these PRs, we have very good recently added test coverage for both local registration & login and also password reset

Reply:
Plus there's a god damn typo in the name! `validatePasssword`

Reply:
ok, I looked at the changes and they seems fine